app/wallet/components/deposit-bottom-sheet.tsx

```diff
@@ -1,15 +1,22 @@
 "use client"
 import { X } from "lucide-react"
 import DepositOptions from "./deposit-options"
+import WithdrawOptions from "./withdraw-options"
 import { Button } from "@/components/ui/button"
 
 interface DepositBottomSheetProps {
   isOpen: boolean
   onClose: () => void
   onDirectDepositClick: () => void
+  operation?: "DEPOSIT" | "WITHDRAW"
 }
 
-export default function DepositBottomSheet({ isOpen, onClose, onDirectDepositClick }: DepositBottomSheetProps) {
+export default function DepositBottomSheet({
```

Comment:
since this component is being shared by deposit and withdrawal, can we please rename this?